docs/components/optimize/userguide/additional-features/variable-labelling.md

```diff
@@ -0,0 +1,20 @@
+---
+id: variable-labeling
+title: Variable Labeling
+description: Add a label for a variable in a process definition.
+---
+
+The variable labeling functionality allows users to add, update and delete batches of variable labels. This allows Optimize to display a variable's label instead of its original name, anywhere the given process definition is being used. Some examples of that would be
```

Comment:
The UI says "rename" and "alias", but not "label". We should try to consolidate terms whenever possible and avoid introducing new ones.
I'm comfortable replacing "alias" with "label" if it improves usability (@asiamalina / @christinaausley )

Reply:
I think Omran has a ticket to update this to "label" everywhere in the UI